frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/BreakoutColumnList/BreakoutColumnListItem/BreakoutColumnListItem.styled.tsx

```diff
@@ -75,19 +75,20 @@ const getSelectedStyle = (theme: Theme) => css`
     color: var(--mb-color-text-white);
   }
 
-  ${BucketPickerPopover.TriggerButton} {
+  .${BucketPickerPopoverStyle.triggerButton} {
+    visibility: visible;
```

Comment:
This shows the label when the column is selected without having to hover over it first.
<img width="472" alt="Screenshot 2025-02-24 at 2 05 46 PM" src="https://github.com/user-attachments/assets/ae1fa9bb-cdd7-4be8-aabd-8a052330e7e3" />